CDCgov/trusted-intermediary
# Complete transfrom coverage in RS e2e tests

# Description

Adds RS e2e test for `ucsdOruRemoveAccessionNumberObservation`
Updates the description of `ucsdOruUpdateUniversalServiceIdentifier` transform to correspond to current usage
Updates the RS e2e test criteria for `UCSD ORU requirements` to use `input.ORC-12` instead of `ORC-12`

## Issue

#1539

## Testing instructions

Run RS e2e tests via Github Actions "Automated Staging Test - Run integration tests". Ensure all tests pass.

## Checklist

- [x] I have added logging where useful (with appropriate log level)
- [x] I have added JavaDocs where required

**Note**: You may remove items that are not applicable
